llvm/unittests/ProfileData/InstrProfTest.cpp

```diff
@@ -374,6 +385,19 @@ IndexedMemProfRecord makeRecord(
   return MR;
 }
 
+IndexedMemProfRecord
+makeRecordV2(std::initializer_list<::llvm::memprof::CallStackId> AllocFrames,
+             std::initializer_list<::llvm::memprof::CallStackId> CallSiteFrames,
+             const MemInfoBlock &Block = MemInfoBlock()) {
+  llvm::memprof::IndexedMemProfRecord MR;
+  for (const auto &CSId : AllocFrames)
+    MR.AllocSites.emplace_back(::llvm::SmallVector<memprof::FrameId>(), CSId,
```

Comment:
Why is the first argument empty?

Reply:
We don't populate `IndexedAllocationInfo::CallStack` because we use it only in `Version0` and `Version1`.